### PR TITLE
OCM-5483 | fix: add cluster arg to help message

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

**Review by robpblake — APPROVED** [review]
/lgtm

File: cmd/create/machinepool/nodepool.go [hunk 1]
```diff
@@ -394,7 +394,8 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		}
 	} else {
 		r.Reporter.Infof("Machine pool '%s' created successfully on hosted cluster '%s'", createdNodePool.ID(), clusterKey)
-		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --machinepool %s'", name)
+		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool  -c %s --machinepool %s'",
```
Comment (ciaranRoche) [inline cmd/create/machinepool/nodepool.go:397]: NP : remove the extra space :pray: 
```suggestion
		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool -c %s --machinepool %s'",
```

File: cmd/create/machinepool/machinepool.go [hunk 2]
```diff
@@ -554,7 +554,8 @@ func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster
 		}
 	} else {
 		r.Reporter.Infof("Machine pool '%s' created successfully on cluster '%s'", name, clusterKey)
-		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --machinepool %s'", name)
+		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool -c %s --machinepool %s'",
```
Comment (ciaranRoche) [inline cmd/create/machinepool/machinepool.go:557]: Would it be better to use the long flag :thinking: `--cluster` 

Probably should look to align this across the CLI

Comment (thomasmckay) [reply cmd/create/machinepool/machinepool.go]: updated

File: cmd/create/machinepool/machinepool.go [hunk 3]
```diff
@@ -556,8 +556,9 @@ func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster
 		}
 	} else {
 		r.Reporter.Infof("Machine pool '%s' created successfully on cluster '%s'", name, clusterKey)
-		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --machinepool %s'", name)
-		r.Reporter.Infof("To view all machine pools, run 'rosa list machinepools -c %s'", clusterKey)
+		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --cluster %s --machinepool %s'",
+			clusterKey, name)
```
Comment (radtriste) [inline cmd/create/machinepool/machinepool.go:560]:
```suggestion
		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool %s --cluster %s'",
			name, clusterKey)
```

File: cmd/create/machinepool/nodepool.go [hunk 4]
```diff
@@ -394,8 +394,9 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		}
 	} else {
 		r.Reporter.Infof("Machine pool '%s' created successfully on hosted cluster '%s'", createdNodePool.ID(), clusterKey)
-		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --machinepool %s'", name)
-		r.Reporter.Infof("To view all machine pools, run 'rosa list machinepools -c %s'", clusterKey)
+		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --cluster %s --machinepool %s'",
+			clusterKey, name)
```
Comment (radtriste) [inline cmd/create/machinepool/nodepool.go:398]:
```suggestion
		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool %s --cluster %s'",
			name, clusterKey)
```

**Review by radtriste — CHANGES_REQUESTED** [review]
There is no `--machinepool` option for `rosa describe machinepool`

```
$ rosa describe machinepool -h
Show details of a machine pool on a cluster.

Usage:
  rosa describe machinepool [flags]

Aliases:
  machinepool, machine-pool

Examples:
  # Show details of a machine pool named "mymachinepool"" on a cluster named "mycluster"
  rosa describe machinepool --cluster=mycluster --machinepool=mymachinepool

Flags:
  -c, --cluster string       Name or ID of the cluster.
  -o, --output string        Output format. Allowed formats are [json yaml]
      --machinepool string   Machine pool of the cluster to target
  -h, --help                 help for machinepool

Global Flags:
      --color string     Surround certain characters with escape sequences to display them in color on the terminal. Allowed options are [auto never always] (default "auto")
      --debug            Enable debug mode.
      --profile string   Use a specific AWS profile from your credential file.
      --region string    Use a specific AWS region, overriding the AWS_REGION environment variable.
```

**Comment by robpblake** [comment]
> --machinepool string   Machine pool of the cluster to target

@radtriste The `rosa describe machine` supports both `--machinepool` or the last parameter as the machine pool name e.g the following are equivalent:

- `rosa describe machinepool -c rblake-test --machinepool rb-test`
- `rosa describe machinepool -c rblake-test rb-test`

There is no shorthand for the `--machinepool` option, but it's clearly listed in the `help` output of the command you pasted. I don't see the need to change this.

cheers,

Rob

**Pull request merged**